plugins/org.locationtech.udig.project.ui/schema/tool.exsd

```diff
@@ -436,6 +436,13 @@ See the command extension point
                </documentation>
             </annotation>
          </attribute>
+         <attribute name="toggling" type="boolean" use="default" value="false">
+            <annotation>
+               <documentation>
+                  Flag indicating whether the toolbar button has toggeling behaviour, i.e. behaves like a switch
```

Comment:
use isToggleButton as a attribute name here ;)

and in documentation is a little type toggeling -> toggling